packages/grafana-ui/src/components/ShareTimeRangeButton/ShareTimeRangeButton.tsx

```diff
@@ -0,0 +1,110 @@
+import { css } from '@emotion/css';
+import { useRef, useState } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+
+import { useStyles2 } from '../../themes';
+import { copyText } from '../../utils/clipboard';
+import { t, Trans } from '../../utils/i18n';
+import { absoluteTimeRangeURL } from '../../utils/time';
+import { Button, ButtonGroup, ButtonProps } from '../Button';
+import { ClipboardButton } from '../ClipboardButton/ClipboardButton';
+import { Dropdown } from '../Dropdown/Dropdown';
+import { Menu } from '../Menu/Menu';
+import { MenuItemElement } from '../Menu/MenuItem';
+
+export interface Props extends ButtonProps {
```

Comment:
maybe we shouldn't extend `ButtonProps` if we don't then do anything with them 🤔